### PR TITLE
REL: create 0.4.0 release notes

### PR DESCRIPTION
**Pull request by adler-j**
Bump the version since the addition of a new top level package (`deform`) should definitely be a new minor version

### CODE REVIEW

File: doc/source/release_notes.rst [hunk 1]
```diff
@@ -9,6 +9,12 @@ Release Notes
 Next release
 ============
 
+ODL 0.4.0 Release Notes (2016-08-17)
+====================================
+
+This release marks the addition of the deform package to ODL, adding functionality for the deformation
+of `DiscreteLp` elements.
```
Comment (kohr-h) [inline doc/source/release_notes.rst]: Double ticks

File: doc/source/release_notes.rst [hunk 2]
```diff
@@ -9,6 +9,12 @@ Release Notes
 Next release
 ============
 
+ODL 0.4.0 Release Notes (2016-08-17)
+====================================
+
+This release marks the addition of the deform package to ODL, adding functionality for the deformation
```
Comment (kohr-h) [inline doc/source/release_notes.rst]: "deform" in double ticks

**Comment by kohr-h** [comment]
ProxImaL coupling missing in release notes.

**Comment by adler-j** [comment]
Fixed comments, merge?

**Comment by kohr-h** [comment]
> ProxImaL coupling missing in release notes.

How about that?

**Comment by kohr-h** [comment]
Go ahead!

**Pull request merged**